src/Microsoft.TemplateSearch.TemplateDiscovery/Program.cs

```diff
@@ -66,8 +68,8 @@ private static async Task Main(string[] args)
                 if (test)
                 {
                     CacheFileTestsBefore60.RunTests(legacyMetadataPath);
```

Comment:
we probably need an issue to create 6.0 tests after 6.0 is GA

Reply:
Created: https://github.com/dotnet/templating/issues/3856